### DISCUSSION TITLE
Should we disclose that we use trademarks?

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#5745] [Discussion]
Since we use trademarks of Scratch in our extension, is it necessary to say "The Scratch name and logo are registered trademarks of the Scratch Foundation"?

I think it'd at least be a good practice.

**Comment by WorldLanguages** [comment]
Do we use the Scratch logo?

**Reply by DNin01** [reply]
We use the **S** from the Scratch logo, but technically, that's just a letter from some font.

**Reply by BroJac5246** [reply]
> We use the **S** from the Scratch logo, but technically, that's just a letter from some font.

But who actually owns that font? Wasn't there an issue that resulted in it being pulled from the costume editor?

**Reply by mybearworld** [reply]
> This font is copyright (c) Jakob Fischer at www.pizzadude.dk,  all rights reserved. Do not distribute without the author's permission.
> 
> Use this font for non-commercial use only! If you plan to use it for commercial purposes, contact me before doing so!

Does this count as commercial?

**Reply by BroJac5246** [reply]
> > This font is copyright (c) Jakob Fischer at www.pizzadude.dk,  all rights reserved. Do not distribute without the author's permission.
> > 
> > Use this font for non-commercial use only! If you plan to use it for commercial purposes, contact me before doing so!
> 
> Does this count as commercial?

[More info on the Scratch Wiki](https://en.scratch-wiki.info/wiki/Scratch_Font)

I suggest that @WorldLanguages or Griffpatch reach out to the font's creator and explain that we'd like to use one letter from one of his fonts, while also pointing out that we don't make a profit and that we're dedicated to making the nonprofit Scratch website easier and better to use. Since our use is already borderline fair use, I would guess that he'd agree.

**Reply by CST1229** [reply]
> > This font is copyright (c) Jakob Fischer at [www.pizzadude.dk](http://www.pizzadude.dk),  all rights reserved. Do not distribute without the author's permission.
> > Use this font for non-commercial use only! If you plan to use it for commercial purposes, contact me before doing so!
> 
> Does this count as commercial?

Probably not, since we aren't making money from SA.
